OptimizelySDK/AudienceConditions/AndCondition.cs

```diff
@@ -38,7 +38,7 @@ public class AndCondition : ICondition
             var foundNull = false;
             foreach (var condition in Conditions)
             {
-                var result = condition.Evaluate(config, attributes, logger);
+                var result = condition.Evaluate(config, user, logger);
```

Comment:
I think we should standardize variable names. `user` itself seems confusing.
UserAttributes userAttributes
userId userId
UserContext userContext

Reply:
Renamed.